modiscolite/util.py

```diff
@@ -2,9 +2,27 @@
 # Authors: Jacob Schreiber <jmschreiber91@gmail.com>
```

Comment:
add yourself to the authors in the next line, if you'd like

Reply:
Done